### PR TITLE
New: enforces PHP 7.2 as minimum version

### PR DESCRIPTION
**Pull request by alexmigf**

### CODE REVIEW

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Comment by alexmigf** [comment]
@MohamadNateqi can you review my last commit please?

File: woocommerce-pdf-invoices-packingslips.php [hunk 1]
```diff
@@ -150,21 +166,16 @@ public function includes() {
...
 			add_action( 'admin_notices', array ( $this, 'need_woocommerce' ) );
 			return;
 		}
-
-		if ( $this->is_woocommerce_version_supported() === false ) {
-			add_action( 'admin_notices', array ( $this, 'need_woocommerce_3_0' ) );
-			return;
-		}
 		
 		// if ( version_compare( PHP_VERSION, '7.2', '<' ) ) {
```
Comment (MohamadNateqi) [inline woocommerce-pdf-invoices-packingslips.php:174]: I think this also can be removed now.

**Review by MohamadNateqi — APPROVED** [review]
All good!

File: woocommerce-pdf-invoices-packingslips.php (hunk 1 above)
Comment (alexmigf) [reply woocommerce-pdf-invoices-packingslips.php]: Done, please check.

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Pull request merged**